sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -36,7 +36,7 @@ export class QueueService {
    * @param queueName queue name
    * @returns queue configuration.
    */
-  private async queueConfigurationDetails(
+  async queueConfigurationDetails(
```

Comment:
If the method accessibility is changed only because of the E2E tests we need to ensure that there is no other way. Ideally, we would not change the "application code" to allow the tests unless necessary.
A possible solution would be to have this configuration updated by the E2E to ensure the expected values are set.
This change can be minor and harmful but can open a door to potentially justify similar changes in the future.

Reply:
It's not only for the E2E tests. It is used on https://github.com/bcgov/SIMS/blob/4a13491825d66a17a86ca8c7664bdf5749fd0d93/sources/packages/backend/apps/queue-consumers/src/processors/schedulers/workflow/assessment-workflow-queue-retry.scheduler.ts#L92

Reply:
This is not the first time that we have implemented a configuration from the DB. I believe that we do not need to use the method you pointed out above if we follow the same process executed for the `process-notifications.scheduler.ts`.

https://github.com/bcgov/SIMS/blob/5854e3d9951f5fdf99d7cbc90f72c03dc285c07e/sources/packages/backend/apps/queue-consumers/src/processors/schedulers/notification/process-notifications.scheduler.ts#L24-L30

Reply:
We can follow the way how polling limit is retrieved for process notifications.

![image](https://github.com/bcgov/SIMS/assets/54600590/59cf796c-ec1d-448e-a20d-669f37daee8c)